src/features/Dummy.js

```diff
@@ -6,16 +6,15 @@ import BiteshareButton from '../components/BiteshareButton.js';
 import { colors } from '../infrastructure/colors.js';
 
 const Title1 = styled.Text`
-color: ${(props) => props.theme.colors.brand.rausch}
-`;
+color: ${(props) => props.theme.colors.brand.rausch}`;
 const Title2 = styled.Text`
   color: ${(props) => props.theme.colors.brand.beach}
 `;
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: 'rgba(243,225,210,0.4)',
```

Comment:
We could probably use `colors.brand.ebisuLight` since `{ colors }` is imported from the shared colors file? or we can add another `ebisu` variation in the shared file and use it here. 

(I would suggest use the shared colors file as much as possible to keep everything consistent lol)

Reply:
I think we can. It's just easy when we are trying out the color by copy and pasting the hex code.
I guess I just need to get into the habit of using the shared colors.